### ISSUE TITLE
Vuforia

### ISSUE THREAD

**Issue by VladIacobIonut** [jiulongw/swift-unity#114]
Hi again,

I have tried to integrate unity in my Xcode Swift project succesfully. However when my Unity project contains Vuforia features as well, the project does not build anymore. The error complains about Vuforia not containing BITCODE.
`You must rebuild it with bitcode enabled (Xcode setting ENABLE_BITCODE), obtain an updated library from the vendor, or disable bitcode for this target. for architecture armv7`

I set ENABLE_BITCODE to false but then more errors appear. I have imported the Data folder as well as creating a reference to the Data/Raw/Vuforia folder as mention in another issue. My version of Unity is 2018.2.18.f1. I am using Xcode 10.

Does anyone have any guess for this problem? Is there any precise solution for this ? 

Thank you very much.

**Comment by alvincrisuy** [comment]
Hello, have you solve your problem?

**Comment by VladIacobIonut** [comment]
Hello, I haven't solved this problem at that point. Ultimately I decided to use the native ARKit instead as it served my usecase just as well as Vuforia.

**Comment by ghus86** [comment]
@alvincrisuy did you find a solution for this ?
@jiulongw do you have some input regarding this issue ?

**Comment by alvincrisuy** [comment]
@ghus86 Yes, I have solved it. What error did you encounter? I encounter a lot 😆

**Comment by ghus86** [comment]
@alvincrisuy Cool. 
I am getting this error about Vuforia not containing BITCODE.

**Comment by alvincrisuy** [comment]
I'll get back to you later as I don't have access to the project now

**Comment by ghus86** [comment]
@alvincrisuy Thanks really need some help with this =/
